### PR TITLE
Fix decoding issue of git version for py3

### PR DESCRIPTION
**Pull request by xylar**
The `utility_scripts` were not included in #177 

Fixes a bug by decoding the `git_version` into a unicode string in `manage_regression_test_suite.py`, required for python 3 compatibility.

### CODE REVIEW

**Comment by xylar** [comment]
## Testing

I used this fix as part of testing my new COMPASS conda environment for python 3 (see https://github.com/MPAS-Dev/geometric_features/pull/102#issuecomment-487390567).

File: testing_and_setup/compass/utility_scripts/compare_fields.py [hunk 1]
```diff
@@ -18,153 +21,153 @@
 args = parser.parse_args()
 
 if not args.filename1:
-	parser.error("Two filenames are required inputs.")
+    parser.error("Two filenames are required inputs.")
```
Comment (xylar) [inline testing_and_setup/compass/utility_scripts/compare_fields.py]: Lots of tabs have been replaced with spaces.  It's a bad practice to have a mix of tabs and spaces in a python file, and generally tabs should be avoided.

**Comment by xylar** [comment]
@mark-petersen, this was included in #229 and was therefore part of your testing and mine.  However, it needs to get merged to `develop` and then we need to merge `develop` to `ocean/develop` before #229 can go in.

File: testing_and_setup/compass/utility_scripts/compare_fields.py (hunk 1 above)
Comment (mark-petersen) [reply testing_and_setup/compass/utility_scripts/compare_fields.py]: Thanks. I appreciate that.

**Comment by mark-petersen** [comment]
@mgduda and @matthewhoffman: FYI this is COMPASS only, but a merge into develop. It does not effect atmosphere core.  I will be merging this today.

**Comment by xylar** [comment]
Thanks!

**Pull request merged**